builder.go

```diff
@@ -94,10 +96,22 @@ func (b *Builder) Build() (*BuildMetadata, error) {
 			return nil, err
 		}
 
+		updateDefaultProcesses(br.Processes, api.MustParse(bpTOML.BuilpackAPI()), b.PlatformAPI)
```

Comment:
You could also call `bp.API` here to avoid having to modify the interface.

Reply:
You're so right. This is what happens when making changes too late. Thanks Natalie, I'll make this change.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/520/commits/63f4e9a470ff92f133caa5ffff56d047d6fa3295.